README.md

```diff
@@ -20,10 +20,7 @@ It is also easy to graph data in this format. The following example is a histogr
 
 ## Installation
 
-1. `nvm use` (if you use NVM)
-2. `npm ci`
-3. `npm run build`
-4. `npm link`
+`npm i -g lighthouse-parade`
```

Comment:
@calebeby We should think about ways to address node version requirement. But I'll create another issue to deal with that.